pywr/notebook/save_graph.js

```diff
@@ -0,0 +1,53 @@
+
+
+require.config({paths: {d3: 'https://d3js.org/d3.v5.min'}});
+  
+require(["d3"], function(d3) {
```

Comment:
Add a comment / docstring

Reply:
Not sure how to format this as the function is an anonymous one passed as a callback to require. The alternative could be to add some more comments to the code within the function?